lib/common/util/map-keys.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import forEach from 'lodash.foreach'
```

Comment:
I'd recommend adding the entire lodash as a dependency. Watch out when importing directly though, camelCase counts when you're importing files. 
```js
import forEach from 'lodash.foreach'
/// becomes
import forEach from 'lodash/forEach'